app/views/declaration/commodity_details.scala.html

```diff
@@ -35,6 +35,27 @@
 @tradeTariffLink = {<a target="_blank" href=@appConfig.tradeTariffUrl>@messages("declaration.commodityDetails.how.tradeTariff")</a>}
 @classificationTeamLink = {<a target="_blank" href=@appConfig.classificationHelpUrl>@messages("declaration.commodityDetails.how.classificationTeam")</a>}
 
+
+
+@tariffBodyDesc = { @Html(messages("declaration.type.consignmentTariffText", components.details_content_link())) }
+
+@tariffBody = {
+@tariffBodyDesc
+<ul class="govuk-list govuk-list--bullet">
```

Comment:
This section is repeated exactly same way in few files - what do you think about static partial for this ?

Reply:
Done